packages/ui-extensions/src/surfaces/checkout/targets.ts

```diff
@@ -734,13 +726,21 @@ export interface RenderExtensionTargets {
     AnyComponent
   >;
   /**
-   * A static extension target that is rendered on top of the **Thank you** page as an overlay.
+   * A static extension target that is rendered on top of the Checkout page as an overlay.
```

Comment:
- [ ] @jffortier made these changes 
- [ ] @jffortier approves these changes for stable `2024-10` 

[Overlay: add thank-you target to TYP](https://github.com/Shopify/checkout-web/commit/ded25b5221feceba7b10129996019dc58974b01b)

Reply:
Okay! i will remove this from the unstable and stable release. Once the stable has been released, it can be released to `unstable`